utils/mil_msgs/action/MoveTo.action

```diff
@@ -3,6 +3,7 @@ Header header
 mil_msgs/PoseTwist posetwist
 float64 speed
 bool uncoordinated # false goes in a straight line, true achieves some components before others
+bool do_waypoint_validation # true does waypoint check, false ignores waypoint check
```

Comment:
I prefer that this be named ```bool blind``` like for [lqRRT](https://github.com/jnez71/lqRRT/blob/master/demos/lqrrt_ros/action/Move.action). Besides being a more concise name, this defaults to false ensuring a move is verified for safety by default.